core/src/test/java/org/apache/iceberg/TestCatalogUtil.java

```diff
@@ -118,7 +118,7 @@ public void loadCustomFileIO_badArg() {
   public void loadCustomFileIO_badClass() {
     AssertHelpers.assertThrows("cannot cast",
         IllegalArgumentException.class,
-        "does not implement FileIO",
+        "does not implement org.apache.iceberg.io.FileIO",
```

Comment:
I think `class.getSimpleName()` could avoid changing this, but leaving as is doesn't harm.

Reply:
thanks, will do that